### DISCUSSION TITLE
Teleport Connect - Desktop App

### DISCUSSION THREAD

**Discussion by karmops** [gravitational/teleport#45533] [Q&A] [answered]
I can't login using passwordless via teleport connect

![image](https://github.com/user-attachments/assets/b99ca20a-9560-4814-b49f-584a163ba016)

**Comment by webvictim** [comment, marked as answer]
I believe this is due to this issue: https://github.com/gravitational/teleport/issues/35770

Essentially "Touch ID in the browser" and "Touch ID at the command line" are two totally separate things in MacOS, so if you register a passwordless user in the browser, you won't be able to access that user from the command line (or Teleport Connect)
